### DISCUSSION TITLE
Papermerge 2.0.0 - use locals folders outside of Docker container

### DISCUSSION THREAD

**Discussion by MareusCalgar** [ciur/papermerge#546] [Q&A]
Dear all,

I am facing the following problem. I can setup and run Papermerge 2.0.0 (i need the automates functions).

However, I want to user folders on my ZFS-Pool to store the files and also create a seperate import folder to which I can scan new pdf files via my network.

The folders I want to implement are:

/zpool/documents <--- to store the documents
/zpool/import <---- import folder for papermerge which will be shared to the network via Samba

I tried several modifications to the docker-compose.yml file and also mounting docker volumes but nothing works. 

Could someone please help me how to edit the docker-compose.yml?

Thank you so much.

**Comment by VadarVillage** [comment]
Check out the docs for "extension fields" in docker-compose: https://docs.docker.com/compose/compose-file/compose-file-v3/#extension-fields


You can map host directories to the named volume like this screenshot. In case you can't see the line i've marked, the syntax is "- /opt/documents:/opt/documents". Obviously those are the directories I chose, but you'll want to add a similar line:

![image](https://github.com/ciur/papermerge/assets/25759247/381b24d1-07c5-47f0-ac0d-6510193eb26c)
